snapshot/snapshot.go

```diff
@@ -716,7 +718,25 @@ func overlayMount(options []string) []mount.Mount {
 		},
 	}
 }
+func (o *snapshotter) passImageNameMounts(ctx context.Context, parent string) ([]mount.Mount, error) {
+	_, pInfo, _, err := snapshot.GetSnapshotInfo(ctx, o.ms, parent)
+	if err != nil {
+		return nil, errors.Wrapf(err, "passImageNameMounts get snapshot %q info failed", parent)
+	}
+	passImageOption := fmt.Sprintf("imageurl=%s", pInfo.Labels["containerd.io/snapshot/cri.image-ref"])
```

Comment:
I suppose it's possible that the key `containerd.io/snapshot/cri.image-ref` does not exist in snapshot's labels. We should handle such case

Reply:
Thanks for review. I have fixed it.